src/taxes/pages/TaxChannelsPage/TaxChannelsPage.tsx

```diff
@@ -53,12 +58,19 @@ interface TaxChannelsPageProps {
   disabled: boolean;
 }
 
+export type TaxCountryConfiguration = Omit<
+  TaxConfigurationPerCountryFragment,
+  "taxCalculationStrategy"
+> & {
+  taxCalculationStrategy: string;
+};
+
 export interface TaxConfigurationFormData {
   chargeTaxes: boolean;
-  taxCalculationStrategy: TaxCalculationStrategy;
+  taxCalculationStrategy: string;
```

Comment:
question:  can we use stronger type here?

Reply:
We can have now multiple values here:
1) `flatRate` from `TaxCalculationStrategy`
2) id of an app
3) plugin id 

That is why I weaken type here